Documentation/UsingSetting/PageTSconfig.rst

```diff
@@ -75,6 +75,72 @@ TYPO3 v11 and v12 by importing the content of this file with the API function
     :language: php
     :caption: EXT:my_sitepackage/ext_localconf.php
 
+
+.. _include-static-page-tsconfig-per-site:
+
+Page TSconfig on site level
+===========================
+
+..  versionadded:: 13.1
+    Page TSconfig can be included on a per site level.
+
+Site Page TSconfig is loaded from :file:`page.tsconfig` if placed next to
+site configuration file :file:`config.yaml` and is scoped to pages within that
+site.
+
+This way sites and sets can ship page TSconfig without the need for database
+entries or by polluting global scope. Dependencies can expressed via site sets,
```

Comment:
What does "polluting global" mean in this case? Adding conditional expressions into a central tsconfig file for "switches"? If thats the case, maybe we can reword it a bit (or else remove the "polluting" part if we can't pinpoint what that means?)

Reply:
I actually copied this from the changelog entry but that is probably what they mean?

Reply:
OK. Before we phrase it wrong, let's leave it at that. It's not recommended to "pollute" anymore, so we must not explain it :D